src/main/resources/templates/db_scripts/schema.json

```diff
@@ -1219,6 +1219,11 @@
       "snippetPath": "publication-period/drop-publication-period-functions.sql",
       "fromModuleVersion": "27.2.0"
     },
+    {
+      "run": "after",
+      "snippetPath": "publication-period/migratePublicationPeriod.sql",
+      "fromModuleVersion": "27.2.0"
```

Comment:
As we introduce breaking change we have to update module version to 28.0.0

Reply:
done